SETUP_ORACLE.md

```diff
@@ -1,5 +1,19 @@
 # Setup Oracle
 
+## Install Oracle dependencies using the script
+
+1. Run the script with sudo
+```shell
+$ sudo ./script/oracle/install-instantclient-packages.sh 
```

Comment:
@jsmadis , btw there are RPMs from oracle that can be used in Fedora or other RPM based distros